classroom.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'studnet'
```

Comment:
- [ ] You don't need to require students in the classroom so it would be better to remove it cause you will just pass students as parameters also it return an error cause you have a typo `studnet`